py/autoevals/oai.py

```diff
@@ -44,7 +46,11 @@ def prepare_openai(is_async=False, api_key=None, base_url=None):
     openai_obj = openai
     is_v1 = False
 
-    if hasattr(openai, "OpenAI"):
+    if client is not None:
+        # v1+ AzureOpenAI and OpenAI clients have a `_version` attribute
+        is_v1 = getattr(client, "_version", "0") >= "1"
```

Comment:
Not in love with this approach, since it's an undocumented API. A couple of options:
* Always set `is_v1` to true for custom clients (and require that they're compatible with `OpenAIWrapper`)
* Set it dynamically based on the `openai` import, even though this _may_ not be the basis for the client, i.e., `is_v1 = hasattr(openai, "OpenAI")`
* Something else...?

Reply:
See your comment and I'll hardcode it.